test/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ruby:2.4.1
+RUN apt-get update && \
+    apt-get install -y jq && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+ENV SSH_KEY id_rsa
+RUN { \
+    echo "#!/bin/bash"; \
+    echo "eval \$(ssh-agent) >/dev/null"; \
+    echo "ssh-add /root/.ssh/\$SSH_KEY"; \
+    echo "exec \"\$@\""; \
+    } > /entrypoint.sh && chmod 755 /entrypoint.sh
+ENTRYPOINT ["/entrypoint.sh"]
+
+RUN mkdir -p /root/.puppetlabs/bolt && \
+    echo "disabled: true" > /root/.puppetlabs/bolt/analytics.yaml
+
+COPY .cdpe-workflow-tests-config.json /root/
```

Comment:
Just FYI... this file does not seem to be the same one that the workflow tests are depending on.

I copied the file out of this directory to use with workflow tests and it not laid out the same way as the tests are expecting. For instance, there should be a `storage` key per:

https://github.com/puppetlabs/PipelinesInfra/blob/master/workflow-tests/tests/install/setupStorageS3.test.js#L12

However, there isn't one.

```
> JSON.parse(fs.readFileSync('cdpe-workflow-tests-config.json')).storage
undefined
```

Instead the settings live under

```
> JSON.parse(fs.readFileSync('cdpe-workflow-tests-config.json')).STORAGE_SETTINGS
{
  S3: {
    ...
  },
  ARTIFACTORY: {
    ...
  }
}
```

Reply:
Ah ah nevermind! You had told me to grab the config values I needed from `artifactory.delivery.puppetlabs.net/cd4pe-maven-test-runner:latest` -- and the details I'm describing are for the file `/root/.cdpe-test-credentials.json`... *not* the file from this reop / container.

The file in this container is what I needed to run workflow tests.

Sorry for the confusion!